fiftyone/core/cli.py

```diff
@@ -2702,6 +2704,372 @@ def _print_operator_info(operator_uri):
+            )
+
+        if args.type == "local":
+            _launch_delegated_local()
+
+
+def _launch_delegated_local():
+    try:
+        dos = food.DelegatedOperation()
+
+        print("Delegated operation server running")
+        print("\nTo exit, press ctrl + c")
+        while True:
+            dos.execute_queued_operations(log=True)
+            time.sleep(1)
+    except KeyboardInterrupt:
+        pass
+
+
+class DelegatedListCommand(Command):
+    """List delegated operations that you've run.
+
+    Examples::
+
+        # List all delegated operations
+        fiftyone delegated list
+
+        # List some specific delegated operations
+        fiftyone delegated list \\
+            --dataset quickstart \\
+            --operator @voxel51/io/export_samples \\
+            --state COMPLETED \\
+            --sort-by COMPLETED_AT \\
+            --limit 10
+    """
+
+    @staticmethod
+    def setup(parser):
+        parser.add_argument(
+            "-o",
+            "--operator",
+            default=None,
+            metavar="OPERATOR",
+            help="only list operations for this operator",
+        )
+        parser.add_argument(
+            "-d",
+            "--dataset",
+            default=None,
+            metavar="DATASET",
+            help="only list operations for this dataset",
+        )
+        parser.add_argument(
+            "-s",
+            "--state",
+            default=None,
+            help=(
+                "only list operations with this state. Supported values are: "
+                "('QUEUED', 'RUNNING', 'COMPLETED', 'FAILED')"
+            ),
+        )
+        parser.add_argument(
+            "--sort-by",
+            default="queued_at",
+            help=(
+                "how to sort the operations. Supported values are: "
+                "('QUEUED_AT', 'STARTED_AT', COMPLETED_AT', 'FAILED_AT', 'OPERATOR')"
+            ),
+        )
+        parser.add_argument(
+            "--reverse",
+            action="store_true",
+            default=None,
+            help="whether to sort in reverse order",
+        )
+        parser.add_argument(
+            "-l",
+            "--limit",
+            type=int,
+            default=None,
+            help="a maximum number of operations to show",
+        )
+
+    @staticmethod
+    def execute(parser, args):
+        dos = food.DelegatedOperation()
+
+        state = _parse_state(args.state)
+        paging = _parse_paging(
+            sort_by=args.sort_by, reverse=args.reverse, limit=args.limit
+        )
+
+        ops = dos.list_operations(
+            operator=args.operator,
+            dataset_name=args.dataset,
+            run_state=state,
+            paging=paging,
+        )
+
+        _print_delegated_list(ops)
+
+
+def _parse_state(state):
+    from fiftyone.operators.executor import ExecutionRunState
+
+    if state is not None:
+        try:
+            state = ExecutionRunState[state.upper()]
```

Comment:
@nebulae I had to add this boilerplate for `run_state`, `sort_by`, `sort_direction` to convert strings to enums. Also elsewhere in the code there has to be `.value` append to enums when the string is desired.

It's old school, but what about implementing "enums" just as classes:

```python
class ExecutionRunState(object):
    RUNNING = "running"
    COMPLETED = "completed"
    ...
```

so that these methods and run docs themselves could just directly use strings?